HW04/Nastja_Kireeva/lib/notification.rb

```diff
@@ -0,0 +1,16 @@
+class Notification
+  attr_reader :message, :homework
+
+  def initialize(homework:)
+    @homework = homework
+    @message = message
+  end
+
+  def send_message
+    message = { new: "#{homework.mentor.fullname} has assigned a new homework for you. See #{homework.title}",
+    to_work: "A student #{homework.student.fullname} takes #{homework.title} to work",
+    to_check: "The homework #{homework.title} is ready to be checked",
+    reject: "#{homework.mentor.fullname} has rejected your homework #{homework.title}",
+    accept: "#{homework.mentor.fullname} has accepted your homework #{homework.title}" }
+  end
```

Comment:
I would create a separate method for each status 

```suggestion
  def send_message
    { new: new_message,
     ...
    accept: accept_message }
  end
  
  def accept_message
    "#{homework.mentor.fullname} has accepted your homework #{homework.title}"
  end
  
```